bubblegum/program/src/lib.rs

```diff
@@ -850,6 +848,12 @@ pub mod bubblegum {
             BubblegumError::TreeAuthorityIncorrect,
         );
 
+        if !authority.contains_mint_capacity(1) {
+            return Err(BubblegumError::InsufficientMintCapacity.into());
+        }
+
+        authority.increment_mint_count();
```

Comment:
Hmm, maybe it's better to move this increment after `process_mint_v1` completes successfully, just so we don't also end up counting mint reqs that end up leading to errors.

Reply:
Whole transaction and state change will be reverted if there's a failure.